src/include/iscsi-client/dialogs.rb

```diff
@@ -272,13 +272,13 @@ def initialize_iscsi_client_dialogs(include_target)
               _("No Authentication"),
               true,
               VBox(
-                Left(Label(_("Incoming Authentication"))),
+                Left(Label(_("Outgoing Authentication"))),
                 HBox(
                   HWeight(2, TextEntry(Id(:user_in), _("Username"))),
```

Comment:
I think that would be fine, only labels are wrong, but the source code are fine, tested